changelogs/3.0.0.md

```diff
@@ -0,0 +1,478 @@
+  immediately stopped with a segmentation fault (gh-9235).
+* Fixed a bug when the assertion in `box_promote_qsync` would fail in the
+  debug build mode. The assertion is that at the moment when `box_promote_qsync`
+  is called, no other promote is being executed. It turned out that this
+  assertion is basically incorrect. In the release build mode, this incorrect
+  assumption could potentially lead to writing 2 PROMOTE entries in the same
+  term (gh-9263).
+
+### Replication
+
+* Fixed a bug that allowed to change replicaset UUID via `_schema` space.
+* Fixed a bug when new instances could try to register via an anon instance
+  which previously failed to apply `box.cfg{replication_anon = false}`.
+* Fixed a bug when `box.ctl.demote()` with `box.cfg{election_mode = 'off'}`
+  and an owned synchro queue could simply not do anything (gh-6860).
+* Fixed anonymous replicas not receiving the synchronous transaction queue state
+  during join (gh-9401).
+* Now transactions are not rolled back if the transaction fiber is
+  cancelled when waiting for quorum from replicas (gh-9480).
+
+### LuaJIT
+
+Backported patches from the vanilla LuaJIT trunk (gh-8825). The following issues
+were fixed as part of this activity:
+
+* Fixed frame for on-trace OOM handling.
+Backported patches from the vanilla LuaJIT trunk (gh-9145). The following issues
+were fixed as part of this activity:
+
+* Fixed error handling after return from a child coroutine.
+* Fixed clashing of addresses in the `__call` metamethod return dispatch (again).
+* Fixed the assembling for the head of the side trace (gh-8767).
+* Prevented compilation of `__concat` methamethod with tailcall to fast
+  function.
+* Fixed buffer overflow in parsing the `#pragma` directive via FFI (gh-9339).
+  Now the error is thrown when more than 6 alignment settings are pushed on the
+  internal stack.
+* Fixed incorrect fold rule for `x - (-0)` on trace (for `x = -0` the result
+  should be `0`).
+* Fixed output for `IR_CONV` in `jit.dump()`.
+* Fixed `math.min()`/`math.max()` inconsistencies for x86/x86_64 architectures
+  when called with a NaN argument or `-0` and `0`.
+* Fixed `math.ceil(x)` result sign for -1 < x < 0.5.
+* Errors from gc finalizers are now printed instead of being rethrown.
+* Fixed `lua_concat()`.
+* Fixed possible storing of NaN keys to table on trace.
+* Fixed ABC FOLD optimization with constants.
+* Marked `CONV` as non-weak, to prevent invalid control flow path choice.
+* Fixed CSE of a `REF_BASE` operand across `IR_RETF`.
+* Fixed the fold rule for `BUFHDR APPEND`.
+* Fixed HREFK, ALOAD, HLOAD, forwarding vs. `table.clear()`.
+* Fixed snapshot PC when linking to `BC_JLOOP` that was a `BC_RET*`.
+* Fixed dangling references to CType.
+* Ensured returned string is alive in `ffi.typeinfo()`.
+* Fixed the missing initialization of the internal structure, leading to a
+  crash when recording a trace with an allocation of cdata.
+
+### Lua
+
+#### Fio
```

Comment:
I would drop the sub-section (fio), because it has just one item.

Reply:
done